docs/DMS/PRIMARY-DATA-STORAGE.md

```diff
@@ -49,7 +49,8 @@ able to hit those goals.
 
 ```mermaid
 erDiagram
-    References }o--|| Aliases : "2 FKs to Aliases per row, parent and referenced"
+    References }o--|| Aliases : "Referenced document alias"
+    References }o--|| Documents : "Parent document of reference"
```

Comment:
The actual change - forgot to update this part of the ER diagram. Other changes are formatting only.